includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -26,7 +26,7 @@ class Plugin_Readme_Check extends Abstract_File_Check {
 	 */
 	protected function check_files( Check_Result $result, array $files ) {
 		// Find the readme file.
-		$readme = self::filter_files_by_regex( $files, '/readme\.txt$/' );
+		$readme = self::filter_files_by_regex( $files, '/readme\.txt|readme\.md$/' );
```

Comment:
This could be shortened:

```suggestion
		$readme = self::filter_files_by_regex( $files, '/readme\.(txt|md)$/' );
```

Aside: looks like this check is not case insensitive? 

The check from the other plugin is case insensitive, see:

https://github.com/WordPress/plugin-check/blob/fd36709abdb126be93cd96979f5e1eefe5661f0a/plugin.php#L76

So we might want to add that "i" as well:

```suggestion
		$readme = self::filter_files_by_regex( $files, '/readme\.(txt|md)$/i' );
```

Reply:
Fixed.